notifications.json

```diff
@@ -0,0 +1,861 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Notification API",
+        "description": "Notification API"
+    },
+    "host": "e2effzjxvtgbmuilw-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/Messages/{id}": {
+            "parameters": [
+                {
+                    "name": "id",
+                    "in": "path",
+                    "required": true,
+                    "type": "string"
+                }
+            ],
+            "get": {
+                "operationId": "getMessage",
+                "summary": "Get Message",
+                "tags": [
+                    "Messages"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/message-output"
+                        }
+                    },
+                    "400": {
```

Comment:
I don't understand, I'm saying we need a description for each status code, see [here](https://swagger.io/docs/specification/describing-responses/) for an example, in the *HTTP Status Codes* section